airflow/contrib/kubernetes/pod_launcher.py

```diff
@@ -74,15 +74,25 @@ def run_pod(self, pod, startup_timeout=120):
                 time.sleep(1)
             self.log.debug('Pod not yet started')
```

Comment:
Would be good to also write to log, the actual state here? In case it's `PodInitializing` or other initializing states?

Reply:
So the actual full status is somewhat verbose, and I'm not sure if it remains consistent. Perhaps return ['conditions']['type']?

```python
{'conditions': [{'last_probe_time': None,
                 'last_transition_time': datetime.datetime(2018, 1, 15, 23, 23, 44, tzinfo=tzutc()),
                 'message': None,
                 'reason': None,
                 'status': 'True',
                 'type': 'PodScheduled'}],
 'container_statuses': None,
 'host_ip': None,
 'init_container_statuses': None,
 'message': None,
 'phase': 'Pending',
 'pod_ip': None,
 'qos_class': 'BestEffort',
 'reason': None,
 'start_time': None}
```

Reply:
Fair point - I think `status.phase` might be the most succinct status that we could write. If you think it doesn't add value, also ok with leaving it as is.

Reply:
I don't think it adds value since the status is guaranteed to be pending or it wouldn't be in the locked state. I'd put that under "let's wait for someone to ask for it"

Reply:
SGTM